static/app/views/dashboards/manage/index.tsx

```diff
@@ -175,15 +239,41 @@ function ManageDashboards() {
   }
 
   function renderDashboards() {
-    const dashboardsPageLinks = getResponseHeader?.('Link') ?? '';
     return (
       <DashboardList
         api={api}
         dashboards={dashboards}
         organization={organization}
-        pageLinks={dashboardsPageLinks}
         location={location}
         onDashboardsChange={() => refetchDashboards()}
+        isLoading={isLoading}
+        rows={rows}
+        columns={columns}
```

Comment:
Nit, but I think `columnCount` and `rowCount` would be better since `rows` implies and array of `Row` objects (whatever that is), and same for `columns`